Documentation/docs-mobile/messages/apt2000.md

```diff
@@ -0,0 +1,29 @@
+---
+title: .NET for Android error/warning APT2000
+description: APT2000 error/warning code
+ms.date: 04/11/2024
+---
+# .NET for Android error/warning APT2000
+
+## Well-known APT2000 error messages
+
+- `error APT2000: Invalid file name: must contain only [a-z0-9_.]`
+- `error APT2000: Resource entry resource_name is already defined.`
+- `error APT2000: No resource found that matches the given name (at 'resource_name' with value '@string/foo').`
+- `error APT2000: invalid resource directory name: obj\Debug\dir with spaces "dir with spaces".`
+- `warning APT2000: warning: string 'resource_name' has no default translation.`
+
+## Issue
+
+This message indicates that `aapt2` (Android Asset Packaging Tool) reported an error or warning. `aapt2` is part of the Android SDK and is used internally by .NET for Android to process and compile resources into binary assets.
+
+Errors reported by `aapt2` are outside of .NET for Android's control, so a general error code of APT2000 is used reporting the exact message coming from `aapt2`.
```

Comment:
Should we link to here, as a place to file bugs for `aapt2`?
* https://developers.google.com/issue-tracker/

Reply:
The gist of this sentence is that this is a catch all for any unknown aapt2 error, that doesn't mean its a bug in aapt2. 
Perhaps I need to reword it,